### PR TITLE
Get projects

### PR DESCRIPTION
**Pull request by LyskaL**
get a list of projects by ID group. 

Also, created User and Group class.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,14 +6,28 @@
...
+			<artifactId>jackson-core</artifactId>
+			<version>2.9.0.pr1</version>
+		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/org.json/json -->
+		<dependency>
+			<groupId>org.json</groupId>
+			<artifactId>json</artifactId>
+			<version>20160810</version>
+		</dependency>
```
Comment (strazhkoe) [inline pom.xml]: Do we still need still need these dependencies?

File: src/main/java/com/ystrazhko/git/entities/Group.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
+package com.ystrazhko.git.entities;
+
+public class Group {
```
Comment (strazhkoe) [inline src/main/java/com/ystrazhko/git/entities/Group.java]: Please, add javadoc

File: src/main/java/com/ystrazhko/git/entities/Group.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
+package com.ystrazhko.git.entities;
+
+public class Group {
+    private final int id;
+    private final String name;
+    private final String path;
+    private final String description;
+    private final boolean lfs_enabled;
```
Comment (strazhkoe) [inline src/main/java/com/ystrazhko/git/entities/Group.java]: Please, remove _ from the middle of the field name. Should be lfsEnabled. Do it in all fields here and in the User class

File: src/main/java/com/ystrazhko/git/entities/Group.java [hunk 4]
```diff
@@ -0,0 +1,84 @@
+package com.ystrazhko.git.entities;
+
+public class Group {
+    private final int id;
```
Comment (strazhkoe) [inline src/main/java/com/ystrazhko/git/entities/Group.java]: Please, start fields from _

File: src/main/java/com/ystrazhko/git/entities/User.java [hunk 5]
```diff
@@ -0,0 +1,138 @@
+package com.ystrazhko.git.entities;
+
+public class User {
```
Comment (strazhkoe) [inline src/main/java/com/ystrazhko/git/entities/User.java]: Please, add javadoc

**Review by strazhkoe — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (LyskaL) [reply pom.xml]: Removed unused dependencies.

File: src/main/java/com/ystrazhko/git/entities/Group.java (hunk 3 above)
Comment (LyskaL) [reply src/main/java/com/ystrazhko/git/entities/Group.java]: If I will change name's fields, then JSONParser will works incorrect.

File: src/main/java/com/ystrazhko/git/entities/Group.java (hunk 4 above)
Comment (LyskaL) [reply src/main/java/com/ystrazhko/git/entities/Group.java]: If I will change name's fields, then JSONParser will works incorrect.

**Review by strazhkoe — APPROVED** [review]

**Pull request merged**